include/TrustWalletCore/TWHarmonyAddress.h

```diff
@@ -0,0 +1,54 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "TWBase.h"
+#include "TWData.h"
+#include "TWString.h"
+
+TW_EXTERN_C_BEGIN
+
+struct TWPublicKey;
+
+/// Represents an Harmony address.
+TW_EXPORT_CLASS
+struct TWHarmonyAddress;
+
+/// Compares two addresses for equality.
+TW_EXPORT_STATIC_METHOD
+bool TWHarmonyAddressEqual(struct TWHarmonyAddress *_Nonnull lhs,
+                           struct TWHarmonyAddress *_Nonnull rhs);
+
+/// Determines if the string is a valid address.
+TW_EXPORT_STATIC_METHOD
+bool TWHarmonyAddressIsValidString(TWString *_Nonnull string);
+
+/// Creates an address from a string representaion.
+TW_EXPORT_STATIC_METHOD
+struct TWHarmonyAddress *_Nullable TWHarmonyAddressCreateWithString(TWString *_Nonnull string);
+
+/// Creates an address from a key hash.
+TW_EXPORT_STATIC_METHOD
+struct TWHarmonyAddress *_Nullable TWHarmonyAddressCreateWithKeyHash(TWData *_Nonnull keyHash);
+
```

Comment:
@hewigovens can we reuse `CosmosAddress`? it's the same bech32 address

Reply:
Cosmos is compressed public key

Reply:
Oh :( then it's ok to keep as is